src/background.js

```diff
@@ -93,21 +94,40 @@ function init() {
 
   getMetricsUUID().then(clientUUID => {
     Metrics.init(clientUUID);
-    return getAlarms();
+    return getNextOpenAlarms();
```

Comment:
Hmm, `getNextOpenAlarms` gets called twice - maybe don't need this first call, since `handleNextOpen` does it again & checks length anyway? Or maybe I'm missing something in the logic here?